ceremonies/src/mock.rs

```diff
@@ -64,7 +64,7 @@ impl dut::Config for TestRuntime {
 	type MeetupMinSize = MeetupMinSize;
 	type MeetupNewbieLimitDivider = MeetupNewbieLimitDivider;
 	type WeightInfo = ();
-	type MaxAttestations = ConstU32<10>;
+	type MaxAttestations = ConstU32<15>;
```

Comment:
as assignments with 19 participants are possible, why don't we set this to >=18, i.e. 20 to be safe?

Reply:
only a test mock, but I'm not sure if this makes tests brittle

Reply:
good catch